js/templates/modals/onboarding/onboarding.html

```diff
@@ -0,0 +1,137 @@
+<section class="contentBox padMd clrP clrBr clrSh3 <%= `${ob.screen}Screen` %>">
+  <div class="flexVCent gutterH posR">
+    <div class="<% if (ob.screen === 'intro') print('flexExpand') %>"><%= ob.brandingBoxT() %></div>
+    <% if (ob.screen === 'intro') { %>
+      <% if (ob.curConn && ob.curConn.server) { %>
+        <% const serverName = ob.curConn.server.get('name') || ''; %>
+        <div><span class="txB"><%= ob.polyT('onboarding.introScreen.connectionLbl') %></span> <%= serverName %></div>
+        <div>
+          <button class="btn clrP js-changeServer"><%= ob.polyT('onboarding.introScreen.btnChange') %></button>
+        </div>
+      <% } %>
+    <% } else if (ob.screen === 'info') { %>
+      <div class="flexExpand txB tx3">
+        <div class="center"><%= ob.polyT('onboarding.infoScreen.heading') %></div>
+      </div>
+      <div class="clrT2 txB"><%= ob.polyT('onboarding.pageOfPage', { startPage: 1, totalPages: 2 }) %></div>
+    <% } else if (ob.screen === 'tos') { %>
+      <div class="flexExpand txB tx3">
+        <div class="center"><%= ob.polyT('onboarding.tosScreen.heading') %></div>
+      </div>
+      <div class="clrT2 txB"><%= ob.polyT('onboarding.pageOfPage', { startPage: 2, totalPages: 2 }) %></div>
+    <% } %>
+  </div>
+  <hr class="clrBr" />
+  <div class="mainContent">
+    <% if (ob.screen === 'intro') { %>
+    <div class="flexCent">
+      <div>
+        <div class="txCtr rowSm">
+          <span class="txUp txB clrT2"><%= ob.polyT('onboarding.introScreen.introLine') %></span><i class="btcIcon margRTn"></i>
```

Comment:
I gave it some space via CSS since adding in an actual space gives it more space than what the design reflects (haven't pushed yet).